contracts/interfaces/IOrderMixin.sol

```diff
@@ -133,12 +134,13 @@ interface IOrderMixin {
     ) external payable returns(uint256 makingAmount, uint256 takingAmount, bytes32 orderHash);
 
     /**
-     * @notice Same as `fillOrder` but allows to specify extensions that are used for the order
+     * @notice Same as `fillOrder` but allows to specify arguments that are used by the taker.
```

Comment:
It might make sense to provide more details about these parameters and their usage; idk, what do you think?